frontend/src/constants/States.ts

```diff
@@ -0,0 +1,31 @@
+export enum headerStates {
```

Comment:
I think it would be best if all types (including enums) were in `PascalCase`. also, I'm used to the values in an `enum` being all `UPPER_CASE`, but what you have is fine since it would require a lot of refactoring